src/Fable.ReactGoogleMaps/Fable.Helpers.ReactGoogleMaps.fs

```diff
@@ -0,0 +1,172 @@
+
+    [<RequireQualifiedAccess>]
+    type InfoWindowProperties =
+    | OnCloseClick of (unit -> unit)
+        interface IInfoWindowProperties
+
+    type ISearchBoxProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type SearchBoxProperties =
+    | Ref of (obj -> unit)
+    | OnPlacesChanged of (unit -> unit)
+        interface ISearchBoxProperties
+
+    type IMarkerProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type MarkerProperties =
+    | Key of obj
+    | Title of string
+    | Icon of string
+    | OnClick of (unit -> unit)
+    | Position of Coordinates.Position
+        interface IMarkerProperties
+
+    type IMarkerClustererProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type MarkerClustererProperties =
+    | Key of obj
+    | AverageCenter of bool
+    | MaxZoom of int
+    | EnableRetinaIcons of bool
+    | GridSize of float
+        interface IMarkerClustererProperties
+
+    type IMapProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type MapProperties =
+    | SetRef of (obj -> unit)
+    | ApiKey of string
+    | DefaultZoom of int
+    | SearchBoxText of string
+    | ShowSearchBox of bool
+    | ShowTrafficLayer of bool
+    | DefaultCenter of Coordinates.Position
+    | Center of Coordinates.Position
+    | OnCenterChanged of (unit -> unit)
+    | OnPlacesChanged of (Places.Place [] -> unit)
+    | OnZoomChanged of (unit -> unit)
+    | OnIdle of (unit -> unit)
+    | Markers of React.ReactElement
+    | MapLoadingContainer of string
+    | MapContainer of string
+        interface IMapProperties
+let InfoWindow: RCom = import "InfoWindow" "react-google-maps"
+
+/// A wrapper around google.maps.InfoWindow
+let infoWindow (props:Props.IInfoWindowProperties list) child : React.ReactElement =
+    R.from InfoWindow (keyValueList CaseRules.LowerFirst props) [child]
+
+let SearchBox: RCom = import "SearchBox" "react-google-maps"
+
+/// A wrapper around google.maps.places.SearchBox on the map
+let searchBox (props:Props.ISearchBoxProperties list) children : React.ReactElement =
+    R.from SearchBox (keyValueList CaseRules.LowerFirst props) children
+
+
+let Marker: RCom = import "Marker" "react-google-maps"
+
+/// A wrapper around google.maps.Marker
+let marker (props:Props.IMarkerProperties list) children : React.ReactElement =
+    R.from Marker (keyValueList CaseRules.LowerFirst props) children
+
+let MarkerClusterer: RCom = import "MarkerClusterer" "react-google-maps/lib/components/addons/MarkerClusterer.js"
+
+/// A wrapper around MarkerClusterer - https://github.com/mahnunchik/markerclustererplus/blob/master/docs/reference.html
+let markerClusterer (props:Props.IMarkerClustererProperties list) (markers:React.ReactElement list) : React.ReactElement =
+    R.from MarkerClusterer (keyValueList CaseRules.LowerFirst props) markers
+
+let GoogleMapComponent: RCom = import "GoogleMapComponent" "./js/mapComponent.js"
```

Comment:
how can we make sure mapComponent.js is bundled? or can we rewrite as fable module?

Reply:
Just add it to the package in the `fable` folder keeping the same directory structure and it will work ([example](https://github.com/MangelMaxime/Thoth/blob/5bbe97540218d992289ca673b9593babafcc23bd/src/Thoth.Json/Thoth.Json.fsproj#L21)).